### PR TITLE
add uppercase and change some messages

### PR DESCRIPTION
**Pull request by giuseppevignanello**
Using the command I noticed that the messages and choices are lowercase in the terminal. They seem better in uppercase. 

I also changed the phrase for Typescript integration to React. 

This is my first contribution to an Open Source project, please be kind if I did something wrong with GitHub 👏

### CODE REVIEW

File: utils/wizard.mts [hunk 1]
```diff
@@ -1,103 +1,109 @@
...
+  template: string;
+  folder: string;
+};
+const wizard = async () => {
+  return inquirer
+    .prompt([
+      {
+        type: "input",
+        name: "componentName",
+        message: "give a name to your component",
```
Comment (Giuliano1993) [inline utils/wizard.mts:15]: Since you're working on it, can you fix it here too? Thank you so much

**Review by Giuliano1993 — CHANGES_REQUESTED** [review]
That's a pretty great fix, thank you so much! Just fix  that one more line and we can get it merged 😄

**Comment by Giuliano1993** [comment]
Thank you for your contribution @giuseppevignanello I really appreciated your work.

Just as a side note, since it is your first time contributing on open-source it's usually a good practice to open an issue first and have it assigned, before you work on a PR ( you can checkout also the [Contributing](https://github.com/Giuliano1993/make-js-component/blob/main/CONTRIBUTING.md) file )

Still, thank you for pointing that out and working on a fix 😄

File: utils/wizard.mts (hunk 1 above)
Comment (giuseppevignanello) [reply utils/wizard.mts:15]: For sure!

Comment (giuseppevignanello) [reply utils/wizard.mts]: Done!

**Comment by giuseppevignanello** [comment]
> Thank you for your contribution @giuseppevignanello I really appreciated your work.
> 
> Just as a side note, since it is your first time contributing on open-source it's usually a good practice to open an issue first and have it assigned, before you work on a PR ( you can checkout also the [Contributing](https://github.com/Giuliano1993/make-js-component/blob/main/CONTRIBUTING.md) file )
> 
> Still, thank you for pointing that out and working on a fix 😄

Good to know, Thank you for the note!

**Review by Giuliano1993 — APPROVED** [review]

**Pull request merged**